src/firefly/js/visualize/ui/CatalogSelectViewPanel.jsx

```diff
@@ -421,25 +458,56 @@ var userChangeDispatch = function () {
                 if (fieldKey === 'project'
                     || fieldKey === 'catalog') {
                     idx = 0; // reset to first item of the catalog list
+                    cleanFilterRestrictions(inFields);
```

Comment:
good catch. That should be fixed.

On 6/20/16 4:03 PM, loitly wrote:
>
> In src/firefly/js/visualize/ui/CatalogSelectViewPanel.jsx 
> <https://github.com/Caltech-IPAC/firefly/pull/108#discussion_r67782648>:
>
> > @@ -421,25 +458,56 @@ var userChangeDispatch = function () {
> >                  if (fieldKey === 'project'
> >                      || fieldKey === 'catalog') {
> >                      idx = 0; // reset to first item of the catalog list
> > +                    cleanFilterRestrictions(inFields);
>
> oversight? no assignment made to the returned value.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Caltech-IPAC/firefly/pull/108/files/85d4c4b40a55416915aaa562b3fd437c012b9e99#r67782648>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AAvATBKas347nfhRKn2EKwwbnioGzCNgks5qNxxDgaJpZM4I5_Xm>.
>

-- 
Emmanuel Joliet
NASA/IPAC  Infrared Science Archive
MS 100-22, Caltech, Pasadena, CA 91125
Office: 170
Email:  ejoliet@ipac.caltech.edu
Phone:  (626) 395-1489